.github/workflows/release-embedding-sdk.yml

```diff
@@ -0,0 +1,173 @@
+name: Release Metabase Embedding SDK for React
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_version:
+        description: 'SDK version (e.g. 0.1.3)'
```

Comment:
idea: we could validate the SDK version in case we mistakenly passed in an invalid version number. not important yet.

Reply:
good point. I'll add a todo comment.

Reply:
fixed in https://github.com/metabase/metabase/pull/42516/commits/d39e5f0e5b8524464105473471a89cda5eb2b4b0